README.md

```diff
@@ -6,33 +6,52 @@ This project demonstrates the use of Euclideon udSDK developer Kit (udSDK SDK) w
 Language:              C#
 Type:                  Integration
 Contributor:           Euclideon udSDK Development Team <support@euclideon.com>
-Organization:          Euclideon, https://euclideon.com/
+Organization:          Euclideon, https://www.euclideon.com/
 Date:                  2020-08-10
-udSDK Version:         1.0.0
+Last Updated:          2021-09-24
+udSDK Version:         2.1.0
 Toolsets:              Requires Unity >= 2019.3.4f1
 ```
 
 ## Quickstart guide 
 
 **The Unity-udSDK sample requires a free Euclideon Account to use, licenses can be obtained [here](https://www.euclideon.com/udsdk/)** 
-The udSDK is tested with Unity 2019.3.4f1 - it may work in other versions of Unity, but we can't guarantee that it does. Please sneure you have Unity 2019.3.4f1 installed.
+The udSDK is tested with Unity on following versions:
+```
+- 2019.3.4f1
+- 2020.1.9f1
+- 2020.3.18f1
+- 2021.1.22f1
+```
+It may work in other versions of Unity, but no guarantee is provided for this. Please ensure you have one of the tested versions installed.
 
 ### Installation - New or Existing Project
 The fastest way to install udSDK for Unity is to go [here](https://Euclideon.com/unity) and follow the onscreen instructions.
```

Comment:
We should probably get rid of this link while we are changing stuff - it's old, not very helpful (the "instructions" it includes point you back to the Github)

Reply:
Done